src/core/mouseEvent.js

```diff
@@ -53,8 +52,8 @@ var allowedEvents = [
  * @param {*} eventOptions
  * @returns {Event}
  */
-var createEvent = function createEvent(eventName, eventOptions) {
-    var event;
+const createEvent = function createEvent(eventName, eventOptions) {
```

Comment:
**quibble:** Converting the code to use `let` and `const` instead of `var` is good, but why not continue and also the correct pattern for functions?

Usually, when you see `var something = function() { ... }`, this is either a developer's taste (which is IMO debatable), or because the function needs to be redefined later on. If the function is immutable, then the best practice is to use a named function instead of `const something = function() { ... }`. Why? because `function` hoists, while `const` is lexically scoped. This also removes redundant and cumbersome naming.

**suggestion:**
```suggestion
function createEvent(eventName, eventOptions) {
```